Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -713,6 +715,18 @@ public void disableSmartSession() {
         prefHelper_.disableSmartSession();
     }
 
+
+    /**
+     * <p>
+     *  Enable Facebook app link check operation during Branch initialisation.
+     * </p>
+     *
+     * @param enableFacebookAppLinkCheck Set with true to enable Facebook app link check operation
+     */
+    public void enableFacebookAppLinkCheck(boolean enableFacebookAppLinkCheck) {
```

Comment:
let's just make it set it to true automatically since it'll be false at runtime by default